### PR TITLE
Update Numi.

### PR DESCRIPTION
**Pull request by jiashuw**
- Remove deprecated appcast.
- Update app url.

### CODE REVIEW

File: Casks/numi.rb [hunk 1]
```diff
@@ -2,9 +2,7 @@
   version :latest
   sha256 :no_check
 
-  url 'http://numi.io/static/files/Numi.zip'
-  appcast 'http://numi.io/update.xml',
-          :sha256 => 'fc6e87987f66f478065ac353a270fa122fb329e85f5b7fd6cb256caefd22b7ed'
+  url 'http://dl.devmate.com/com.dmitrynikolaev.numi/beta/Numi.zip'
```
Comment (vitorgalvao) [inline Casks/numi.rb]: Needs a `# devmate.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

**Comment by vitorgalvao** [comment]
Why is the appcast deprecated? What makes you think that?

**Comment by jiashuw** [comment]
@vitorgalvao 
The latest Numi is `3.0b6` but it's not reflected in anyway in its original [appcast](http://numi.io/update.xml).

**Comment by vitorgalvao** [comment]
Looks good, then. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git) and @mention me after?

**Comment by jiashuw** [comment]
@vitorgalvao good to go :)

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as ecfd5e7c6e5498172a8c53cc0fe83b334dbfcab8 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**